ui/src/components/ChipComponent/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import Chip from '@material-ui/core/Chip';
+import { withStyles } from '@material-ui/core/styles';
+
+@withStyles(() => ({
+  chipLabel: {
+    whiteSpace: 'nowrap',
+    userSelect: 'text',
+    cursor: 'inherit',
+  },
+  chipRoot: {
+    whiteSpace: 'nowrap',
+    userSelect: 'text',
+    cursor: 'inherit',
+  },
+}))
+export default class ChipComponent extends Component {
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <Chip
+        classes={{ label: classes.chipLabel, root: classes.chipRoot }}
+        label={this.props.label}
+        className={this.props.className ? this.props.className : null}
```

Comment:
Because of the comment I just wrote up, I couldn't simply replace ChipComponent everywhere. 
One way to export this every where could be by using ternary operators. 
I can create ternary for all possible props for Chips used through the Taskcluster UI (there aren't many - so this option is not too unfeasible)

Other than that, I'm always open to suggestions from you! (Thanks as always ;) )